From 613411b33351c58f4ed71cd3e9b6a2fb09bd2d64 Mon Sep 17 00:00:00 2001
From: Joshua Strobl <joshua@streambits.io>
Date: Tue, 14 Jul 2020 12:19:12 +0300
Subject: [PATCH 1/1] Support a stateless configuration.

Remove /usr/etc since in no circumstance would that ever be supported in Solus.
---
 src/conf/alsa.conf | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/conf/alsa.conf b/src/conf/alsa.conf
index a04173a..12e8fe1 100644
--- a/src/conf/alsa.conf
+++ b/src/conf/alsa.conf
@@ -8,10 +8,10 @@
 	{
 		func load
 		files [
-			"/usr/etc/alsa/conf.d"
 			"/etc/alsa/conf.d"
-			"/etc/asound.conf|||/usr/etc/asound.conf"
+			"/etc/asound.conf"
 			"~/.asoundrc"
+			"/usr/share/defaults/alsa/asound.conf"
 			{
 				@func concat
 				strings [
-- 
2.27.0

